rclcpp/include/rclcpp/serialized_message.hpp

```diff
@@ -77,6 +77,29 @@ class RCLCPP_PUBLIC_TYPE SerializedMessage : public rcl_serialized_message_t
 
   /// Destructor for a SerializedMessage
   virtual ~SerializedMessage();
+
+  /// Get the underlying rcl_serialized_t handle
+  rcl_serialized_message_t * get();
+
+  // Get a const handle to the underlying rcl_serialized_message_t
+  const rcl_serialized_message_t * get() const;
```

Comment:
Since you are storing an instance (and not a pointer) to the rcl struct, can you not return a reference in these getters?

Reply:
Also, I'd use the fuller name `get_rcl_serialized_message()`, which follows other existing classes doing something similar, e.g.:

https://github.com/ros2/rclcpp/blob/0f0a4a8e397720ace85299eb30592e710b1b339b/rclcpp/include/rclcpp/init_options.hpp#L54-L57